tests/services/label.service.spec.ts

```diff
@@ -1,100 +1,50 @@
-      githubService.fetchAllLabels.and.callFake(() => of([]));
-      of(true).pipe(labelService.syncLabels(false)).subscribe();
-
-      assertLabelCreated(githubService, LabelConstant.SEVERITY_LOW_LABEL);
-      assertLabelCreated(githubService, LabelConstant.TYPE_DOCUMENTATION_BUG_LABEL);
-      expect(githubService.createLabel).toHaveBeenCalledTimes(LabelService.getRequiredLabelsAsArray(false).length);
-    });
-
-    it('should create missing required labels for team response phase if some required labels are fetched', () => {
-      githubService.fetchAllLabels.and.callFake(() => of(LabelConstant.SOME_TEAM_RESPONSE_PHASE_LABELS));
-      of(true).pipe(labelService.syncLabels(true)).subscribe();
-
-      assertLabelNotCreated(githubService, LabelConstant.SEVERITY_LOW_LABEL);
-      assertLabelCreated(githubService, LabelConstant.RESPONSE_REJECTED_LABEL);
-      assertLabelCreated(githubService, LabelConstant.STATUS_DONE_LABEL);
-      assertLabelCreated(githubService, LabelConstant.TYPE_DOCUMENTATION_BUG_LABEL);
-      expect(githubService.createLabel).toHaveBeenCalledTimes(
-        LabelService.getRequiredLabelsAsArray(true).length - LabelConstant.SOME_TEAM_RESPONSE_PHASE_LABELS.length
-      );
-    });
-
-    it('should create missing required labels for tester phase if some required labels are fetched', () => {
-      githubService.fetchAllLabels.and.callFake(() => of(LabelConstant.SOME_TESTER_PHASE_LABELS));
-      of(true).pipe(labelService.syncLabels(false)).subscribe();
-
-      assertLabelNotCreated(githubService, LabelConstant.SEVERITY_HIGH_LABEL);
-      assertLabelCreated(githubService, LabelConstant.TYPE_FUNCTIONALITY_BUG_LABEL);
-      expect(githubService.createLabel).toHaveBeenCalledTimes(
-        LabelService.getRequiredLabelsAsArray(false).length - LabelConstant.SOME_TESTER_PHASE_LABELS.length
-      );
-    });
-
-    it('should not need to create any required labels if all required labels are fetched', () => {
-      githubService.fetchAllLabels.and.callFake(() => of(LabelConstant.ALL_REQUIRED_LABELS_ARRAY));
-      of(true).pipe(labelService.syncLabels(true)).subscribe();
-
-      expect(githubService.createLabel).toHaveBeenCalledTimes(0);
-    });
-  });
-});
 
 describe('LabelService: parseLabelData()', () => {
   beforeAll(() => {
     labelService = new LabelService(null);
-    labelList = labelService.parseLabelData(LabelConstant.SOME_TEAM_RESPONSE_PHASE_LABELS);
+    labelList = labelService.parseLabelData(LabelConstant.SEVERITY_LABELS);
   });
 
   afterAll(() => {
     labelService = null;
   });
 
-  it('should be response.Accepted label', () => {
-    expect(labelList[0].labelCategory).toBe(LabelConstant.RESPONSE);
-    expect(labelList[0].labelValue).toBe(LabelConstant.RESPONSE_ACCEPTED);
-    expect(labelList[0].labelColor).toBe(LabelConstant.COLOR_RESPONSE_ACCEPTED);
+  it('should be severity very low label', () => {
+    expect(labelList[0].category).toBe(LabelConstant.CATEGORY_SEVERITY);
```

Comment:
After the merging of #254, `Label` does not have the `category` and `formattedName` properties